### PR TITLE
Note Making Feature Added: Issue #35

### PR DESCRIPTION
**Pull request by plusminuschirag**
Note Making Feature is added. **With Reference to issue #35** 

[Follow this link for setting up Groot for note making feature](https://github.com/ugroot/GROOT/issues/35)

### CODE REVIEW

**Comment by BhanuPrakashNani** [comment]
Can you send a pic of the note feature here?

**Comment by plusminuschirag** [comment]
Screenshot from 2019-04-12 17-35-55
Yeah Have a Look Please.

**Comment by BhanuPrakashNani** [comment]
Nice work chirag.

**Comment by BhanuPrakashNani** [comment]
Can we have options as we have in Google keep - to have lists with checkbox. Something like that.

**Comment by iamchiragsharma** [comment]
I will try my best. But I guess creating another issue for that would be great.

**Comment by plusminuschirag** [comment]
Let's create another issue for that.

**Comment by omi10859** [comment]
@BhanuPrakashNani have you tested the code on your local machine?

**Comment by BhanuPrakashNani** [comment]
I am testing it now. Sorry for the delay.

**Comment by BhanuPrakashNani** [comment]
To test the pull request I need permissions to merge it. As I need to test it in the command line, I should be able to see the option "Merge Pull Request", beside that there is an option to test it in the command line. Please give me permission to merge so that I can test the pull request.

**Comment by plusminuschirag** [comment]
Issue is reviewed by @BhanuPrakashNani is it ready to be merged now?

**Comment by BhanuPrakashNani** [comment]
I am not able to test the pull request as I mentioned before. I need permission to do so.

**Comment by omi10859** [comment]
@BhanuPrakashNani already told you clone @iamchiragsharma his repo switch to branch test it. You don't need any permissions.

File: requirements/requirements-linux.txt [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 beautifulsoup4==4.7.1
 bs4==0.0.1
 certifi==2018.11.29
 chardet==3.0.4
 Click==7.0
 gTTS==2.0.3
 gTTS-token==1.1.3
-idna==2.8
-isort==4.3.10
+idna==2.5
```
Comment (omi10859) [inline requirements/requirements-linux.txt:21]: any particular reason for changing the versions?

File: requirements/requirements-linux.txt [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
 PyQt5-sip==4.19.14
-requests==2.21.0
+requests==2.17.1
 ruamel.yaml==0.15.89
 six==1.12.0
 soupsieve==1.8
 SpeechRecognition==3.8.1
 typed-ast==1.2.0
-urllib3==1.24.1
+urllib3==1.21.1
```
Comment (omi10859) [inline requirements/requirements-linux.txt:53]: why 24 --> 21 ?

**Review by omi10859 — APPROVED** [review]
I'm approving the changes while there are some things that you should keep in mind.
Groot should be made in such a way so the user can use it/with over voice commands only

File: requirements/requirements-linux.txt (hunk 1 above)
Comment (plusminuschirag) [reply requirements/requirements-linux.txt:21]: There was conflict on the package version requirement by old packages and new ones needed for the further functions.

File: requirements/requirements-linux.txt (hunk 2 above)
Comment (iamchiragsharma) [reply requirements/requirements-linux.txt]: I don't remember it exactly but one of the new packages works only with the 1.21.1 version.

**Comment by plusminuschirag** [comment]
@BhanuPrakashNani  Please Look into the changes and approve them.

**Review by BhanuPrakashNani — APPROVED** [review]
PR is perfectly working. Tested and reviewed on my system. Perfect implementation of Wikimedia API. Nice work @iamchiragsharma.

**Pull request merged**